docs/resources/Channel.md

```diff
@@ -1150,7 +1160,53 @@ Creates a new thread that is not connected to an existing message. The created t
 
 \*\* The 3 day and 7 day archive durations require the server to be boosted. The [guild features](#DOCS_RESOURCES_GUILD/guild-object-guild-features) will indicate if that is possible for the guild.
 
-\*\*\* In API v9, `type` defaults to `PRIVATE_THREAD` in order to match the behavior when thread documentation was first published. In API v10 this will be changed to be a required field, with no default.
+\*\*\* In API v9, `type` defaults to `GUILD_PRIVATE_THREAD` in order to match the behavior when thread documentation was first published. In API v10 this will be changed to be a required field, with no default.
+
+## Start Thread in Forum Channel % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/threads
+
+Creates a new thread in a forum channel, and sends a message within the created thread. Returns a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) on success, and a 400 BAD REQUEST on invalid parameters. Fires a [Thread Create](#DOCS_TOPICS_GATEWAY/thread-create) and [Message Create](#DOCS_TOPICS_GATEWAY/message-create) Gateway event.
+
+- The type of the created thread is `GUILD_PUBLIC_THREAD`.
```

Comment:
This phrasing makes it seem like you don't need to specify type 11 when using this endpoint or that you can't have private threads in a forum. Unfortunately not passing `type` leads to a 50024 error. I'm unsure if that's a bug.

Reply:
Pretty sure this endpoint is broken atm, if you do include the type field, it will ignore the message data, and if you don't it'll give you an error.

Reply:
This is one of those fun, but rare, times where the docs are _ahead_ of the code lol.
Route isn't live yet, and I'll need to migrate our callers over to it gracefully, so for at least a month, our clients will have to opt-in to the new behavior, and then once updates are fully available everywhere I'll switch the new behavior to be the default for forums and remove the old one.